### ISSUE TITLE
Stylus Libraries

### ISSUE THREAD

**Issue by jscheeler** [krakenjs/kraken-js#338]
I am trying to use Jeet and Nib with Stylus.  Currently, using any Stylus imports to libraries such as these these only work in the production environment after a `grunt build`.  Has anyone figured out how to get these to compile in development? 

I have tried duplicating the kraken-devtools-stylus plugin an appending these libraries following the [Stylus API](http://learnboost.github.io/stylus/docs/js.html#usefn) `use` method, but I still get css compile errors. 

Here is my module js file
```
var lib = require('stylus');
var jeet = require('jeet');
var nib = require('nib');

module.exports = function (options) {
    options.ext = options.ext || 'styl';

    return function styl(data, args, callback) {
        var config = {
            filename: args.context.filePath,
            paths: args.paths,
            use: [nib, jeet]
        };
        lib.render(data.toString(), config, callback);
    };
};
```

Here is the devtools section of my development config file
```
"devtools": {
    "enabled": true,
    "priority": 35,
    "module": {
        "name": "kraken-devtools",
        "arguments": [
            "path:./public",
            "path:./.build",
            {
                "template": {
                    "module": "kraken-devtools/plugins/dustjs",
                    "files": "/templates/**/*.js",
                    "base": "templates",
                    "i18n": "config:i18n"
                },
                // "css": {
                //     "module": "kraken-devtools/plugins/stylus",
                //     "files": "/css/**/*.css"
                // },
                "css": {
                    "module": "../../middleware/stylus",
                    "files": "/css/**/*.css"
                },
                "browserify": {
                    "module": "kraken-devtools-browserify",
                    "files": "/js/**/*.js"
                },
                "copier": {
                    "module": "kraken-devtools/plugins/copier",
                    "files": "**/*"
                }
            }
        ]
    }
}
```

As you can see, I've commented out the original kraken-devtools stylus plugin and replaced the module page with my own, which is stored in `./middleware`

If anyone has had success getting this to work in development, I'd appreciate any guidance!

**TL;DR**
Need some guidance on using Stylus Libraries such as Nib, Autoprefixer, and Jeet in development using middleware. So far this is only working when editing the stylus grunt task and running the app in production.

**Comment by jasisk** [comment]
Hey @jscheeler.

Just gave this a shot, locally, and I think you were _extremely_ close to finding the right answer.

`use: [nib, jeet]` => `use: [nib(), jeet()]`

I made a [tiny little example repo](https://github.com/jasisk/stylus-example) demonstrating it.

That said, this certainly belongs in the official stylus plugin for devtools. [I've opened an issue to that effect](https://github.com/krakenjs/kraken-devtools/issues/45).

Closing this but feel free to continue the conversation here if it still doesn't seem to be working or in the devotees issue if you have any more ideas.

**Comment by jscheeler** [comment]
Ah, I love it when it's something so simple! I also had to update the `css` key in my development.json config file to `stylus` and added the `ext` property as well to that object. 

Thanks for your help @jasisk
